spec/granite_orm_spec.cr

```diff
@@ -12,4 +13,18 @@ describe Granite::ORM::Base do
     t = Todo.new(name: "Elorest")
     t.name.should eq "Elorest"
   end
+
+  it "should provide a to_h method" do
+    t = Todo.new(name: "test todo", priority: 20)
+    result = { "name" => "test todo", "priority": 20 }
+
+    t.to_h.should eq result
+  end
+
+  it "should provide a to_json method" do
+    t = Todo.new(name: "test todo", priority: 20)
+    result = "{\"name\":\"test todo\",\"priority\":20}"
```

Comment:
How about `%({"name":"test todo","priority":20})`?

Reply:
I like this better than escaping all the quotes